### PR TITLE
pull req3

### PR DESCRIPTION
**Pull request by bhavsar-kevin**
test

### CODE REVIEW

File: GitTestApplication/GitTestApplication/Employee.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
     <title>Employee</title>
+    <title>Employee1</title>
```
Comment (akshaypatel823) [inline GitTestApplication/GitTestApplication/Employee.html:5]: Not Required.

**Review by akshaypatel823 — COMMENTED** [review]
Change it.

File: GitTestApplication/GitTestApplication/Employee.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ﻿<!DOCTYPE html>
 <html>
 <head>
+    <title>Employee</title>
```
Comment (bhavsar-kevin) [inline GitTestApplication/GitTestApplication/Employee.html:4]: changes done

**Comment by akshaypatel823** [comment]
Done.

**Pull request merged**